servant-docs/src/Servant/Docs/Internal.hs

```diff
@@ -262,8 +270,8 @@ data Action = Action
 -- 'combineAction' to mush two together taking the response, body and content
 -- types from the very left.
 combineAction :: Action -> Action -> Action
-Action c h p n m ts body resp `combineAction` Action c' h' p' n' m' _ _ _ =
-        Action (c <> c') (h <> h') (p <> p') (n <> n') (m <> m') ts body resp
+Action a c h p n m ts body resp `combineAction` Action _ c' h' p' n' m' _ _ _ =
```

Comment:
If we do left-biased merge of `authInfo`, the comment above should be updated. (But note that, unlike body and content-type, this would probably not be the correct behaviour.)